### PR TITLE
Feat/dep updates 050324

### PR DESCRIPTION
**Pull request by Tristano8**

### CODE REVIEW

**Review by JackKelly-Bellroy — COMMENTED** [review]
This implies support for GHC 9.8. Have you tested with it? If so, can you try updating `tested-with` and regenerating CI?

**Comment by Tristano8** [comment]
Oh i thought 4.19 was 9.6 - i'll test now (or let CI do it)

**Comment by Tristano8** [comment]
The update to [wai-handler-hal-example](https://github.com/bellroy/wai-handler-hal-example/pull/7/files) made the same bounds change - should that also have been tested with 9.8?

**Comment by JackKelly-Bellroy** [comment]
Probably yeah

**Review by JackKelly-Bellroy — APPROVED** [review]

File: flake.nix [hunk 1]
```diff
@@ -21,6 +21,7 @@
           "ghc92"
           "ghc94"
           "ghc96"
+          "ghc98"
```
Comment (JackKelly-Bellroy) [inline flake.nix:24]: I assume you were able to build/get a shell for this

Comment (Tristano8) [reply flake.nix:24]: I will tell you in 18 hours. CI gives me some confidence at least

Comment (JackKelly-Bellroy) [reply flake.nix:24]: I'm happy if it evaluates, everything else is gravy really

**Pull request merged**